README.md

```diff
@@ -5,18 +5,23 @@ If you know M-BoF, you don't need to know anything else.
 
 0. Install Vagrant - https://www.vagrantup.com/
 0. Start Vagrant
-   - `cd hacks_mbof`
+   - `cd ocellus`
    - `vagrant up`
    - `vagrant ssh`
 0. Install Bower packages
    - `cd /vagrant/mbofui`
    - `bower install`
 
 # Development
-0. Initialize and start app server
-    - `cd /vagrant`
```

Comment:
Changing to be specific is good, but we'll need to `cd /vagrant` for a later step.  (starting the server)